apps/src/code-studio/header.js

```diff
@@ -349,22 +337,8 @@ header.showMinimalProjectHeader = function() {
   getStore().dispatch(showMinimalProjectHeader());
 };
 
-// Levelbuilder-only UI for saving changes to a level.
 header.showLevelBuilderSaveButton = function(getChanges) {
-  var projectName = $('<div class="project_name_wrapper header_text">')
-    .append(
-      $('<div class="project_name header_text">').text(
-        'Levelbuilder: edit start code'
```

Comment:
at this point, how are react and non-react DOM elements coexisting in the header? it looks like react is rendering `.header_text` and then this init method is appending to it, but this seems fragile unless there is some way of preventing react from re-rendering and destroying this button.